### PR TITLE
Add Java version to smoke test

### LINKED ISSUE #2596: Dragonwell smoke tests failed as test job number is zero
Dragonwell smoke tests failed as test job number is zero.

https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-linux-aarch64-dragonwell_SmokeTests/4/console

This is a common case when new impl or new vendor JDK is enabled no tests are enabled for them. To fix this  `java -version` test will be added as a universal test job.

### PR DESCRIPTION
**Pull request by sophia-guo**
Fix #2596 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Test:
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-linux-x64-dragonwell_SmokeTests/7/console

**Review by M-Davies — APPROVED** [review]

**Comment by adamfarley** [comment]
LGTM

**Review by adamfarley — APPROVED** [review]

File: test/functional/buildAndPackage/playlist.xml [hunk 1]
```diff
@@ -14,6 +14,16 @@
 -->
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>Java_Version</testCaseName>
+		<command>$(JAVA_COMMAND) -version; $(TEST_STATUS)</command>
```
Comment (llxia) [inline test/functional/buildAndPackage/playlist.xml:19]: Please add `$(JVM_OPTIONS)`. It will prevent any problem in case we add variation in the future.

Comment (sophia-guo) [reply test/functional/buildAndPackage/playlist.xml:19]: If new variation is added should separate specific command line tests be added?

Comment (llxia) [reply test/functional/buildAndPackage/playlist.xml:19]: No, if it is just adding new variations, there is no need to add separate tests. Please see https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/openjdk/playlist.xml#L19

**Review by smlambert — APPROVED** [review]

**Pull request merged**